src/main/kotlin/no/nav/klage/oppgave/domain/oppgavekopi/Status.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.klage.oppgave.domain.oppgavekopi
+
+enum class Status {
+
+    OPPRETTET,
+    AAPNET,
+    UNDER_BEHANDLING,
+    FERDIGSTILT,
+    FEILREGISTRERT;
+
+    companion object {
+        fun getIdFor(status: Status): Long {
+            return when (status) {
+                OPPRETTET -> 1L
+                AAPNET -> 2L
+                UNDER_BEHANDLING -> 3L
+                FERDIGSTILT -> 4L
+                FEILREGISTRERT -> 5L
```

Comment:
Var jo litt styr for å få tak i et enum sin id. Burde/kunne det vært konstruktør i stedet?

`enum class Status(val id: Long) {
   OPPRETTET(1l), etc.
}

`

Reply:
Ja, det er meget mulig.. Jeg kopierte denne koden fra Oppgave. Men jeg burde sikkert skrive den om.. :)

Reply:
Pushet endringer